ddtrace/profiling/serverless_scheduler.py

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+import logging
+
+import attr
+
+from ddtrace.internal import compat
+from ddtrace.internal.utils import attr as attr_utils
+from ddtrace.profiling import scheduler
+
+
+LOG = logging.getLogger(__name__)
+
+
+@attr.s
+class ServerlessScheduler(scheduler.Scheduler):
+    _interval = attr.ib(factory=attr_utils.from_env("DD_SERVERLESS_PROFILING_UPLOAD_INTERVAL", 1, int))
```

Comment:
1. is `SERVERLESS` a prefix commonly used?
2. this is misnamed this is not the upload interval but just the wake up interval. Do we want this to be configurable?

Reply:
Removed.